package.json

```diff
@@ -31,6 +31,7 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.0",
+    "eslint-plugin-tailwindcss": "^3.17.5",
```

Comment:
@MeganSteenkamp Worth considering this dependency instead - it's maintained by Tailwind

https://tailwindcss.com/docs/editor-setup#automatic-class-sorting-with-prettier

Reply:
@tom-blake I did initially look at that. It would mean we would need to configure prettier and then have some duplicate rules like using single quotes or you get conflicting. I feel like having all eslint is a big plus ➕